src/components/FileHistory/styles.styl

```diff
@@ -0,0 +1,35 @@
+.HistoryRowCircleWrapper 
+    width: 2.6rem;
+    height: 2.6rem;
+    justify-content: center;
+    display: flex;
+    align-self: start;
+
+.HistoryRowCirle
+    border 1px var(--silver) solid
+    background-color #fff
+
+.HistoryRowRevisionLoader
+    display flex
+    justify-content center
+
+.HistoryRowCaption
+    margin-bottom 32px
+    padding-left 32px
+    padding-right 32px
```

Comment:
I'm annoyed by this because it's the sort of things we shouldn't be duplicating, but there's no easy way to access this in cozy-ui at the moment. I'll create an issue.

Reply:
I think it would be possible to 

```stylus
@require "cozy-ui/stylus/utilities/spaces.styl"

.HistoryCaption
   margin-bottom spacing_values["xl"]
   ...
```

Reply:
done here : https://github.com/cozy/cozy-ui/pull/1245/files#diff-a086ae4a41c0a3b616e8a8dd55f70794R19